lms/static/js/student_account/views/account_settings_view.js

```diff
@@ -0,0 +1,41 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone',
```

Comment:
Note that there are some JS "linting" errors in at least this file-- the comma after 'backone' is not necessary, and a semicolon is missing on line 40. It should be possible to run JSHint on your code-- see https://github.com/edx/edx-platform/pull/3272. JS linting is integrated in the IDE I use (PyCharm), so I haven't actually tried the steps outlined in #3272.